02_ShowUser.php

```diff
@@ -55,56 +66,65 @@
 
 		<!-- Title for query: show users -->
 		<div style="margin-left:8%">
-			<div class="w3-container w3-dark-grey">
+			<div class="container3">
 				<h2>Show User</h2>
 			</div>
 
-			<br>
-			<!-- Users php -->
-			<div class="w3-row">
-				<div class="w3-third w3-center">
-					<h5><user1>USERS</user1></h5>
-					<h6>
-						<?php
-						//connect.php (tells where to connect servername, username, password, dbaseName)
-						require "communityproject_mysqli.php";
-
-						//create a variable to store sql code for the 'display all users' query
-						$query = ("SELECT * FROM users");
-
-						//run the query
-						$result = mysqli_query($conn,$query);
-
-						while ($output = mysqli_fetch_array($result))
-						{
-							echo "<user2>". $output['User_ID']."</user2><br>";
-							//closes the while loop
-						}
-						?>
-					</h6>
-				</div>
 
+			<div class="container4 w3-display-middle">
+
+				<div class="scroller w3-center w3-row">
+					<div class="w3-cell-row" style="width:100%">
+
+						<!-- Users php -->
+						<div class="w3-container w3-white w3-cell">
```

Comment:
added another container: within the orange one, to create a nice design and border with the orange